tests/Doctrine/Tests/DBAL/Functional/Driver/OCI8/OCI8ConnectionTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Functional\Driver\OCI8;
+
+use Doctrine\DBAL\Driver\OCI8\Driver;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+class OCI8ConnectionTest extends DbalFunctionalTestCase
+{
+    /**
+     * @var \Doctrine\DBAL\Driver\OCI8\OCI8Connection
+     */
+    protected $driverConnection;
+
+    protected function setUp()
+    {
+        if (! extension_loaded('oci8')) {
+            $this->markTestSkipped('oci8 is not installed.');
+        }
+
+        parent::setUp();
+
+        if (! $this->_conn->getDriver() instanceof Driver) {
+            $this->markTestSkipped('oci8 only test.');
+        }
+
+        $this->driverConnection = $this->_conn->getWrappedConnection();
+    }
+
+    /**
+     * @group DBAL-2595
+     */
+    public function testLastInsertIdAcceptsFqn()
+    {
+        try {
+            $this->_conn->executeUpdate('DROP TABLE DBAL2595');
+        } catch (\Exception $e) {
```

Comment:
Should I omit this catch block? Because in theory the test can only be run once per suite, no?